piksi_tools/console/settings.yaml

```diff
@@ -2107,6 +2107,7 @@
   default value: ''
   readonly: false
   Description: Override the device used for cell modem connectivity. If left empty, uses default device discovery to determine the correct device to use.
+  Notes: The cell modem daemon must be disabled in order to change this setting.
```

Comment:
remove daemon, something like "cell modem enable must be set to false..." possibly?